addons/website/views/website_templates.xml

```diff
@@ -2120,16 +2120,19 @@
     1,2,3
 </template>
 
+<!-- PAGE 404 -->
 <template id="page_404" name="Page Not Found">
     <t t-call="http_routing.404">
-        <div class="container o_not_editable">
-            <div class="alert alert-info mt32">
-                <p>This page does not exist, but you can create it as you are administrator of this site.</p>
-                <a role="button" class="btn btn-primary js_disable_on_click post_link" t-attf-href="/website/add/#{ path }#{ from_template and '?template=%s' % from_template }">Create Page</a>
+        <div class="bg-100 pt-1">
+            <div class="container o_not_editable">
+                <div class="alert alert-info mt32 d-flex justify-content-between align-items-center">
+                    <p class="m-0 text-primary">This page does not exist, but you can create it as you are editor of this site.</p>
+                    <a role="button" class="btn btn-primary js_disable_on_click post_link" t-attf-href="/website/add/#{ path }#{ from_template and '?template=%s' % from_template }">Create Page</a>
+                </div>
+                <div class="text-center"><i class="fa fa-info-circle"></i> Edit the content below this line to adapt the default <strong>Page not found</strong> page.</div>
             </div>
-            <div class="text-center text-muted">Edit the content below this line to adapt the default "page not found" page.</div>
```

Comment:
What do you think?

Or something else?
<img width="1734" alt="Screenshot 2020-12-02 at 11 58 29" src="https://user-images.githubusercontent.com/7629949/100864139-e283ec00-3495-11eb-98fd-8e2fe12a64cb.png">

Reply:
What about this?

![image](https://user-images.githubusercontent.com/43962154/100999952-913f2f80-355d-11eb-878a-118fc5e9f508.png)

Reply:
Nevermind.
Maybe, just put back the `.text-muted` to make a visual difference with the editable content.

Reply:
Hmm not so sure about this. Maybe it works better without the `alert alert-info` classes.

![image](https://user-images.githubusercontent.com/43962154/101022573-d9684d00-3571-11eb-860d-b8b3b0f19df9.png)

![image](https://user-images.githubusercontent.com/43962154/101022768-1fbdac00-3572-11eb-8747-3e250c0fa7c5.png)

Reply:
Sorry, it was not clear 😁 
Yes, remove the alert box and put back the muted style.